ModeratorFrontEnd/AIForOrcas/AIForOrcas.Server/Controllers/MetricsController.cs

```diff
@@ -65,35 +65,38 @@ public async Task<IActionResult> GetSystemMetrics([FromQuery] MetricsFilterDTO q
 				metrics.Timeframe = queryParameters.Timeframe;
 
 				// Build base queryable
-				var queryable = await BuildQueryableAsync(queryParameters.Timeframe);
+				var queryable = BuildQueryableAsync(queryParameters.Timeframe);
 
 				// If not metrics to return
 				if (queryable == null || queryable.Count() == 0)
 				{
 					return NoContent();
 				}
 
+				var results = queryable
+					.Select(x => DetectionProcessors.ToDetection(x)).ToList();
```

Comment:
Is ToList necessary here? (see relevant comments in DetectionProcessors.cs below)